wodles/vuls/vuls.py

```diff
@@ -0,0 +1,295 @@
+    elif update_deb or os_name == 'debian':
+        debug('Updating Debian OVAL database...')
+        update_oval('debian', update_os_version) #7 8 9 10
+    elif update_orac or os_name == 'oracle':
+        debug('Updating Oracle OVAL database...')
+        update_oval('oracle', update_os_version) #5 6 7
+
+def main(argv):
+
+    # Minimum CVSS for reporting
+    cvss_min = 0
+    # CVSS source
+    cvss_source=''
+    # Message header
+    header = '1:Wazuh-VULS:'
+    # Notify message header
+    notify_header = '9:Wazuh-VULS:'
+    # Show packages info
+    package_info = 1
+    # Minimum antiquity
+    antiquity_limit = 0
+    # Update databases
+    nvd_year = 2002
+    update_nvd = 0
+    update_rh = 0
+    update_ub = 0
+    update_deb = 0
+    update_orac = 0
+    autoupdate = 0
+    update_os_version = ''
+    only_update = 0
+
+    try:
+        opts, args = getopt.getopt(argv,'h',["mincvss=","updatenvd", "nvd-year=", "updaterh", "updateub", "updatedeb", "updateorac", "autoupdate", "os-version=", "disable-package-info", "antiquity-limit=", "debug", "source=", "onlyupdate"])
+    except getopt.GetoptError:
+        help()
+        sys.exit(2)
+    for opt, arg in opts:
+        if opt == '--updatenvd':
+            update_nvd = 1
+        elif opt == '--nvd-year':
+            nvd_year = int(arg)
+        elif opt == '--updaterh':
+            update_rh = 1
+        elif opt == '--updateub':
+            update_ub = 1
+        elif opt == '--updatedeb':
+            update_deb = 1
+        elif opt == '--updateorac':
+            update_orac = 1
+        elif opt == '--autoupdate':
+            autoupdate = 1
+        elif opt == '--onlyupdate':
+            only_update = 1
+        elif opt == '--os-version':
+            update_os_version = arg
+        elif opt == '--disable-package-info':
+            package_info = 0
+        elif opt == '--antiquity-limit':
+            antiquity_limit = int(arg)
+        elif opt == '--mincvss':
+            cvss_min = float(arg)
+        elif opt == '--source':
+            if arg == 'nvd' or arg == 'oval':
+                cvss_source = arg
+            else:
+                help()
+                sys.exit()
+        elif opt == '--debug':
+            global enable_debug
+            enable_debug = 1
+        elif opt == '-h':
+            help()
+            sys.exit()
+        else:
+            print('Error: Invalid parameter')
+            help()
+            sys.exit(1)
+
+    update(update_nvd, update_rh, update_ub, update_deb, update_orac, '', update_os_version)
+
+    if only_update:
+        sys.exit(0)
+
+    msg = {}
+    msg['event'] = 'Starting vulnerability scan.'
+    send_msg(wazuh_queue, notify_header, msg)
+
+    call([vuls_bin, 'scan', '-results-dir={0}'.format(vuls_log), '-config={0}'.format(vuls_config), '-log-dir={0}'.format(vuls_log)])
+
+    # Extracts the log header
+    data = json.load(open('{0}/current/localhost.json'.format(vuls_log)))
```

Comment:
You should load the JSON data after the call to `vuls report`, in other way no CVE data will be present.